cpp/tests/CMakeLists.txt

```diff
@@ -499,8 +499,8 @@ ConfigureTest(CORE_NUMBER_TEST cores/core_number_test.cpp)
 # - Core Number tests -----------------------------------------------------------------------------
 ConfigureTest(K_CORE_TEST cores/k_core_test.cpp)
 
-###################################################################################################
-# - K-truss tests ---------------------------------------------------------------------------------
+###############################################################################################
+# - K-truss tests --------------------------------------------------------------------------
```

Comment:
Sorry for nitpicking but these two lines shorter than other tests. The end of ####s and ----s should match with other tests.